### PR TITLE
191016-congshaoliyu

### PR DESCRIPTION
**Pull request by GUThe**
#285 ready for review

### CODE REVIEW

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+# 其中生姜可以略多一些这样能压住鱼的腥味
+You can add a bit more gingers to neutralize the fishy odor  
+
+0:01:46.560,0:01:48.560
+# 准备仔姜1小块切成姜丝备用
+Julienne a small chunk of tender ginger for later use
+
+0:01:59.940,0:02:01.940
+# 准备几个二荆条切碎备用
+Chop up a couple of Erjintiao(Erjintiao Chilli Pepper) and set aside
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Chop up a couple of "Erjintiao" chilli pepper and set aside
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+0:02:02.200,0:02:04.260
+# 加二荆条的目的是增加鲜辣味
+The purpose of this step is to boost the Umami & Spicy flavor 
+
+0:02:14.640,0:02:16.640
+# 准备适量的小米辣切碎备用
+MInce some Xiaomila(Xiaomi Chilli) and set aside
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Mince some "Xiaomila" chilli and set aside
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+# 不能吃辣的同学可以适当的减少用量
+You can use less Xiaomila if spiciness is not for you
+
+0:02:23.240,0:02:25.240
+# 准备小葱1把切成小段备用
+Cut some spring onions into short segments and set aside
+
+0:02:31.280,0:02:33.240
+# 最后准备酸菜2片切碎备用
+Shred some Pickled Cabbages, then set aside
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Shred some "Suan Cai" (Chinese pickles), then set aside
```
https://en.wikipedia.org/wiki/Suan_cai
考虑将译法加入对译表？

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+# 辅料准备好之后开始腌制鲫鱼
+Start marinating the carps 
+
+0:02:39.660,0:02:41.980
+# 将鲫鱼表面切出一字花刀备用
+First, cut the surface of the Carp in a horizontal pattern 
+
+0:02:42.200,0:02:45.140
+# 切记刀口不能太深否则鲫鱼下锅容易煮烂
+Remember not to give deep gashes so the carps do not get overcooked
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Remember not to give deep gashes so the carps do not break down while cooking
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+# 最后准备酸菜2片切碎备用
+Shred some Pickled Cabbages, then set aside
+
+0:02:37.180,0:02:39.180
+# 辅料准备好之后开始腌制鲫鱼
+Start marinating the carps 
+
+0:02:39.660,0:02:41.980
+# 将鲫鱼表面切出一字花刀备用
+First, cut the surface of the Carp in a horizontal pattern
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
First, make crosswise slices into the carps
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+0:02:56.260,0:02:58.700
+# 加入适量的料酒拌匀腌制5分钟备用
+Add some cooking wine and mix evenly, marinate for 5 minutes
+
+0:03:05.500,0:03:06.800
+# 下一步开始制作
+Next step, start cooking
+
+0:03:07.640,0:03:09.260
+# （缺失）
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
# 首先我们把锅烧热
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+First, heat the wok
+
+0:03:09.360,0:03:11.360
+# 锅烧热之后加入适量的熟菜籽油（约60克）
+When the wok is hot, add about 60 grams of cooked rapeseed oil
+
+0:03:19.680,0:03:22.600
+# 油温6成热之后加入一半的姜蒜末和酸菜下锅
+Add half of the ginger, garlics and pickled cabbages when the oil is 60 percent done
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Add half of the ginger & garlic mince and suan cai when the oil is 60 percent hot 
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+# 然后将酸菜和料头翻炒几下炒出香味
+Stir-fry the auromatics until the fragrance comes out 
+
+0:03:34.940,0:03:36.940
+# 料头炒香之后加入适量的清水
+Add some water
+
+0:03:37.260,0:03:39.520
+# 清水的量不能太多否则容易抢味
+Do not add to much water or the flavor might be diluted
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Do not add too much water or the flavor might be diluted 
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+# 加入味精1小勺
+A small spoon of MSG
+
+0:03:54.940,0:03:55.980
+# 加入少许白糖
+Add some sugar
+
+0:03:56.620,0:03:58.180
+# 加入适量的生抽酱油（约6克）
+Add some light soy sauce(about 6 grams)
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Add some light soy sauce (about 6 grams)
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+# 调好色之后将处理好的鲫鱼依次下入锅中
+Add the marinated water into the wok
+
+0:04:21.440,0:04:23.560
+# 鲫鱼下锅之后转小火烧6分钟
+Braise the carps on low heat for 6 minutes
+
+0:04:23.820,0:04:26.260
+# 将鲫鱼烧熟烧至入味为止
+The purpose of this step is to cook the carps until it pick up the flavor
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
The purpose of this step is to cook the carps until it picks up the flavor  
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 11]
```diff
@@ -0,0 +1,205 @@
...
+# 鲫鱼下锅之后转小火烧6分钟
+Braise the carps on low heat for 6 minutes
+
+0:04:23.820,0:04:26.260
+# 将鲫鱼烧熟烧至入味为止
+The purpose of this step is to cook the carps until it pick up the flavor  
+
+0:04:27.360,0:04:29.700
+# 6分钟之后将鲫鱼捞出放入盘中备用
+After 6 minutes, drain the carps and put it on a plate
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
After 6 minutes, take out the carps and put them on a plate
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 12]
```diff
@@ -0,0 +1,205 @@
...
+
+0:04:41.340,0:04:43.560
+# 然后将准备好的姜蒜辣椒依次下锅
+Add the ginger, garlics and chilli peppers into the the wok
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Add the ginger, garlic and chilli peppers into the the wok
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 13]
```diff
@@ -0,0 +1,205 @@
...
+
+0:04:54.340,0:04:56.340
+# 再加入适量的水淀粉把汤汁收浓
+Add some starch slurry to shrink the broth
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Add some starch slurry to thicken the broth
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 14]
```diff
@@ -0,0 +1,205 @@
...
+
+0:05:02.860,0:05:05.700
+# 汤汁收浓之后将切好的小葱下锅烧至断生即可
+After thickening the broth, add the spring onions into the wok, brasie it till Duansheng(partially cooked)
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
After thickening the broth, add the green onions into the wok, braise it till "Duansheng" (partially cooked)
```
spring onion 是大葱

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 15]
```diff
@@ -0,0 +1,205 @@
...
+
+0:05:14.540,0:05:16.780
+# 最后将汤汁淋在鲫鱼上面即可食用
+Last step, pour the broth on the carps. The dish is now ready to serve.
+
+0:05:25.020,0:05:27.460
+# 一道非常美味的葱香鲫鱼就制作完成
+A very delicious Spring Onion Braised Carps is finished.
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
A very delicious Green Onion Braised Carps is finished.
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 16]
```diff
@@ -0,0 +1,205 @@
...
+
+0:05:28.740,0:05:30.340
+# 下面开始技术总结
+ Technical Summary of Spring Onion Braised Carps
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
 Technical Summary of Green Onion Braised Carps  
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 17]
```diff
@@ -0,0 +1,205 @@
...
+# 一道非常美味的葱香鲫鱼就制作完成
+A very delicious Spring Onion Braised Carps is finished.
+
+0:05:28.740,0:05:30.340
+# 下面开始技术总结
+ Technical Summary of Spring Onion Braised Carps  
+
+0:05:30.600,0:05:34.440
+# 第一，鲫鱼最好现杀现做，这样的味道才足够鲜美
+First, It is better to cook with freshly butchered Carps to preserve the umami flavor
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
First, it is better to cook with freshly butchered carps to preserve the umami flavor 
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 18]
```diff
@@ -0,0 +1,205 @@
...
+# 下面开始技术总结
+ Technical Summary of Spring Onion Braised Carps  
+
+0:05:30.600,0:05:34.440
+# 第一，鲫鱼最好现杀现做，这样的味道才足够鲜美
+First, It is better to cook with freshly butchered Carps to preserve the umami flavor 
+
+0:05:34.680,0:05:38.060
+# 第二，最后成菜之后可以再泼上花椒油增加麻味
+Second, after finish cooking the dish, you can sprinkle some Sichuan Peppercorn Oil for the numbing flavor
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Second, after finishing cooking the dish, you can sprinkle some Sichuan Peppercorn Oil for the numbing flavor
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 19]
```diff
@@ -0,0 +1,205 @@
...
+
+0:05:38.300,0:05:40.940
+# 第三，此菜为鲜辣味最好不加豆瓣酱
+Third, the flavor profile of this dish is Umami & Numbing, it is better to not add doubanjiang
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Third, the flavor profile of this dish is umami & spicy. It is better to not add doubanjiang
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 20]
```diff
@@ -0,0 +1,205 @@
...
+
+0:05:41.100,0:05:44.160
+# 第四，烧鱼最好能略微咸一些这样才够味
+Forth, add more soy sauce can bake the flavor into the carps when braising
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Forth, adding more soy sauce can make the flavor into the carps when braising 
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 21]
```diff
@@ -0,0 +1,205 @@
...
+# 第三，此菜为鲜辣味最好不加豆瓣酱
+Third, the flavor profile of this dish is Umami & Numbing, it is better to not add doubanjiang
+
+0:05:41.100,0:05:44.160
+# 第四，烧鱼最好能略微咸一些这样才够味
+Forth, add more soy sauce can bake the flavor into the carps when braising 
+
+0:05:44.160,0:05:45.720
+# 葱烧鲫鱼的技术总结完毕
+End of the technical summary of Spring Onion Braised Carps
```
Comment (immoonancient) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
End of the technical summary of Green Onion Braised Carps
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 22]
```diff
@@ -0,0 +1,205 @@
+# https://youtu.be/ZurD6odu08Q
+
+0:00:00.480,0:00:02.480
+# 首先我们准备小鲫鱼10条
+First, prepare 10 carps
+
+0:00:02.740,0:00:04.740
+# 在鱼嘴中插入两根筷子
+Stick two chopsticks in the carps' mouth
```
Comment (Yingzhang1122) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Stick two chopsticks into the carps' mouth
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 23]
```diff
@@ -0,0 +1,205 @@
...
+# 料头炒香之后加入适量的清水
+Add some water
+
+0:03:37.260,0:03:39.520
+# 清水的量不能太多否则容易抢味
+Do not add to much water or the flavor might be diluted 
+
+0:03:49.120,0:03:50.960
+# 烧开之后开始调味
+Boil the water on high heat, after that, start seasoning
```
Comment (Yingzhang1122) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Boil the water on high heat and start seasoning 
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 24]
```diff
@@ -0,0 +1,205 @@
...
+# 加入少许白糖
+Add some sugar
+
+0:03:56.620,0:03:58.180
+# 加入适量的生抽酱油（约6克）
+Add some light soy sauce(about 6 grams)
+
+0:03:58.440,0:04:01.980
+# 同学们鱼肉适量的多加点生抽咸一些更入味
+It is ok to add some more soy sauce for the fish to absorb flavor
```
Comment (Yingzhang1122) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
It is ok to add some more soy sauce for the fish to absorb the flavor
```

File: subtitles/wang-gang/191014-congshaoliyu.sbv [hunk 25]
```diff
@@ -0,0 +1,205 @@
...
+# 同学们鱼肉适量的多加点生抽咸一些更入味
+It is ok to add some more soy sauce for the fish to absorb flavor 
+
+0:04:02.600,0:04:04.600
+# 最后再加入少许老抽调色
+Add some dark soy sauce for coloring 
+
+0:04:10.260,0:04:12.840
+# 调好色之后将处理好的鲫鱼依次下入锅中
+Add the marinated water into the wok
```
Comment (Yingzhang1122) [inline subtitles/wang-gang/191014-congshaoliyu.sbv]:
```suggestion
Put the marinated water into the wok
```

**Pull request merged**